src/exo/API_scheduling.py

```diff
@@ -862,6 +948,10 @@ def reorder_stmts(proc, block_cursor):
 
     args:
         block_cursor    - a cursor to a two statement block to reorder
+        rc              - bool (whether to return relevant cursors)
+
+    relevant cursors:
+        block           - a block containing the two statements after the reordering.
```

Comment:
This is not specific to `reorder_stmts`, but rather for the broader API design: 

I would prefer the names of the fields in the returned data-structure to be as minimal as possible to make the user-code as simple as possible. In this example, I would rather name it `block` rather than `block_cursor`. The `_cursor` suffix generally wouldn't give me any new information. However, this is conflicts with the API naming convention for parameters; parameters are often called `{name}_cursror`. I would argue that the parameters should be renamed to just `{name}` since they generally accept a broader set of arguments than just cursors: patterns. You could argue that the pattern itself is somewhat a cursor, but I am not sure if it is clearer to suffix the parameters with `_cursor`.

Reply:
I agree that the return names don't need "_cursor" because we are only returning cursors. Currently undecided on the scheduling operation arguments, but since those were never consistent to begin with, I don't think we need to worry about conflicting with it?

Reply:
I'm supportive of this change.  Although we should have a better scheme for "type" annotation / argument processors.

Let's defer changing those names to a later PR focused on revamping the argument processors.  As is, the longer parameter names rarely appear in code.